### PR TITLE
basic tests setup

### PR DESCRIPTION
**Pull request by maradondt**
## Describe your changes
- Wrote unit tests for helpers

## Issue ticket number and link
#459 
## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [x] I have tested on a mobile device
- [ ] It has been deployed to people-test.sphinx.chat and tested by others

### CODE REVIEW

File: frontend/app/package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "@loomhq/record-sdk": "^2.35.16",
     "@material/react-material-icon": "^0.15.0",
     "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
+    "@testing-library/react": "^14.0.0",
     "@testing-library/user-event": "^7.1.2",
```
Comment (kevkevinpal) [inline frontend/app/package.json:32]: it might make sense to move these to dev dependcies

**Comment by maradondt** [comment]
@kevkevinpal yes, I think we can use jest config for that, like there https://jestjs.io/docs/configuration#:~:text=Jest%20will%20fail%20if%3A,40%25%20branch%20or%20statement%20coverage.

**Comment by kevkevinpal** [comment]
> @kevkevinpal yes, I think we can use jest config for that, like there https://jestjs.io/docs/configuration#:~:text=Jest%20will%20fail%20if%3A,40%25%20branch%20or%20statement%20coverage.

yea that looks good to me

**Review by kevkevinpal — APPROVED** [review]
LGTM 🚀

**Pull request merged**